private_include/protocol.h

```diff
@@ -31,7 +31,8 @@ extern "C" {
 
 #define MD5_SIZE 32
 
-typedef enum __attribute__((packed))
```

Comment:
Why is `__attribute__((packed))` being dropped for the pragma?

Reply:
This is a duplicate of https://github.com/espressif/esp-serial-flasher/pull/102#discussion_r1535737581

Reply:
Seems like the same issue yeah, didn't compile on MSVC